src/app/ipv/tests/handleJourneyPageRequest.test.ts

```diff
@@ -0,0 +1,257 @@
+import { NextFunction } from "express";
+import { expect } from "chai";
+import sinon from "sinon";
+import proxyquire from "proxyquire";
+import { AxiosResponse } from "axios";
+
+describe("handleJourneyPageRequest", () => {
+  const testReq = {
+    id: "1",
+    url: "/ipv/page",
+    params: { pageId: "prove-identity-no-photo-id" },
+    session: {
+      ipvSessionId: "ipv-session-id",
+      ipAddress: "ip-address",
+      currentPage: "prove-identity-no-photo-id",
+      save: sinon.fake.yields(null),
+    },
+    log: { info: sinon.fake(), error: sinon.fake() },
+    csrfToken: sinon.fake(),
+    i18n: { t: () => "Some label" },
+  } as any;
+
+  const res = {
+    status: sinon.fake(),
+    redirect: sinon.fake(),
+    send: sinon.fake(),
+    render: sinon.fake(),
+    log: { info: sinon.fake(), error: sinon.fake() },
+    locals: { contactUsUrl: "contactUrl", deleteAccountUrl: "deleteAccount" },
+  } as any;
+  let next: NextFunction;
+
+  const coreBackServiceStub = {
+    getProvenIdentityUserDetails: sinon.spy(),
+  };
+
+  const middleware: typeof import("../middleware") = proxyquire(
+    "../middleware",
+    {
+      "../../services/coreBackService": coreBackServiceStub,
+    },
+  );
+
+  beforeEach(() => {
+    res.render = sinon.fake();
+    next = sinon.fake() as any;
+    coreBackServiceStub.getProvenIdentityUserDetails = sinon.spy();
+  });
+
+  context("handling page-ipv-reuse journey route", () => {
+    const nameTestCases = [
```

Comment:
It feels wrong to have all these cases here - I think we should cover the mapping edge cases in `reuseHelper.test.ts`, and keep this focused on the behaviour from the middleware.